src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -1370,8 +1370,6 @@ const Authenticate = (() => {
                 showCTAButton('document', 'pending');
                 break;
             case 'suspected':
-                $('#unverified').setVisibility(1);
-                break;
             case 'rejected':
                 if (Number(submissions_left) < 1) {
                     $('#limited_poi').setVisibility(1);
```

Comment:
![Screen Shot 2022-03-09 at 5 51 09 PM](https://user-images.githubusercontent.com/84661147/157455937-8f44334f-d9dc-4145-b4cb-1154e28f5537.png)

In Deriv App , we have same logic for poi "suspected" and poi "rejected" cases. we are displaying the rejection reasons in both status. so making the same logic here in Binary